enos/enos-scenario-autopilot.hcl

```diff
@@ -38,13 +39,24 @@ scenario "autopilot" {
   }
 
   step "build_vault" {
-    module = matrix.builder == "crt" ? module.build_crt : module.build_local
+    module = "build_${matrix.build}"
 
     variables {
-      build_tags  = var.vault_local_build_tags != null ? var.vault_local_build_tags : local.build_tags[matrix.edition]
-      bundle_path = local.bundle_path
-      goarch      = matrix.arch
-      goos        = "linux"
+      build_tags            = var.vault_local_build_tags != null ? var.vault_local_build_tags : local.build_tags[matrix.edition]
```

Comment:
I see why you want `try()`. Maybe we should cut an enos release before we merge this :D

Reply:
(note: I just released 0.0.12 with `try()`)

Reply:
updated to use try